src/protocol/codec/message.c

```diff
@@ -332,7 +332,6 @@ z_result_t _z_push_body_decode(_z_push_body_t *pshb, _z_zbuf_t *zbf, uint8_t hea
     switch (_Z_MID(header)) {
         case _Z_MID_Z_PUT: {
             pshb->_is_put = true;
-            pshb->_body._put = (_z_msg_put_t){0};
```

Comment:
What the reason to remove this initialization?

Reply:
Can we guarantee that decode will overwrite all fields?

Reply:
Decode will not overwrite all fields (for instance it's not going to write a timestamp if the messaged doesn't have one) but the point of the change is that put message is part of a bigger generic container,`z_network_message_t` which itself is already initialized.